cfgov/jinja2/v1/about-us/careers/application-process/index.html

```diff
@@ -61,13 +61,12 @@
         }}
     </div>
     <section class="job-applications-interfaces">
-        <h2>Different interfaces you may have to use</h2>
+        <h2>What You Will See</h2>
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/1959/commits/7ff8f47f54c20f54b3a7a9ecd5c602add6a99404